applications/openshift/etcd/etcd_unique_ca/rule.yml

```diff
@@ -5,30 +5,31 @@ prodtype: ocp4
 title: 'Configure A Unique CA Certificate for etcd'
 
 description: |-
-    A unique and different CA certificate should be created for <tt>etcd</tt>.
-    To ensure the <tt>etcd</tt> service is using a unique certificate,
-    set <tt>ETCD_TRUSTED_CA_FILE</tt> to <tt>/etc/etcd/ca.crt</tt>
-    in <tt>/etc/etcd/etcd.conf</tt> on the master node that does NOT match
-    the OpenShift CA certificate:
-    <pre>ETCD_TRUSTED_CA_FILE=/etc/ssl/etcd/ca.crt</pre>
+  A unique and different CA certificate should be created for <tt>etcd</tt>.
```

Comment:
nit: this paragraph says 'unique and different' and also 'different and separate', I think we can just say separate in these cases

Reply:
That came over from the previous content.